### PR TITLE
Update core_screenshot_config.yaml

### PR DESCRIPTION
**Pull request by hmhoffman**
Using viewport width 1400 for CO so part of the page isn't cut off.

### CODE REVIEW

File: screenshots/configs/core_screenshot_config.yaml [hunk 1]
```diff
@@ -208,6 +215,10 @@ secondary:
     file: pdf
     message: downloading PDF for TN secondary
 
+  TX:
```
Comment (julia326) [inline screenshots/configs/core_screenshot_config.yaml:218]: you might need to pull your local copy of the repo so that the TX change gets integrated?

**Comment by julia326** [comment]
@hmhoffman sorry to revisit this so late - is this change still necessary?

**Comment by hmhoffman** [comment]
I think so! I don't think I pulled my copy of the repo though, so I will do that now.

**Comment by julia326** [comment]
Integrating this change in the screenshots repo and closing this one here, thank you @hmhoffman !

**Pull request closed without merging**